### PR TITLE
edits that solve warnings: zoom property in css, minor code touches

### PR DESCRIPTION
**Pull request by dhdeangelis**
- replaced deprecated "zoom: 1" by "transform: scale(1)"
- deleting initial indent and a comment in some css solved MIME warnings

### CODE REVIEW

File: content/home.html [hunk 1]
```diff
@@ -110,7 +110,8 @@ <h3><a href="https://sourceforge.net/p/pdl/news/">Recent News</a></h3>
 <script type="text/javascript">
 	// Start slideshow plugin.
 	$('.slideshow').cycle({
-		fx: 'fade,curtainX,zoom,turnDown,scrollRight',
+// 		fx: 'fade,curtainX,zoom,turnDown,scrollRight',
```
Comment (mohawk2) [inline content/home.html:113]: Please don't comment code, just replace it. We have git!

File: css/jquery-ui-1.8.2.custom.css [hunk 2]
```diff
@@ -376,8 +376,8 @@ input.ui-button { padding: .4em 1em; }
 .ui-buttonset { margin-right: 7px; }
 .ui-buttonset .ui-button { margin-left: 0; margin-right: -.3em; }
 
-/* workarounds */
-button.ui-button::-moz-focus-inner { border: 0; padding: 0; } /* reset extra padding in Firefox */
+/*/* workarounds */
+button.ui-button::-moz-focus-inner { border: 0; padding: 0; } /* reset extra padding in Firefox */*/
```
Comment (mohawk2) [inline css/jquery-ui-1.8.2.custom.css:380]: The effect of this is to double-comment something. Please if you want to delete something, actually just delete it :-)

File: google-search.css [hunk 3]
```diff
@@ -1,110 +1,141 @@
...
+
+.gs-webResult.gs-result a.gs-title:visited b {
     color: #950000;
   }
-  .gs-webResult.gs-result a.gs-title:hover,
-  .gs-webResult.gs-result a.gs-title:hover b {
+
+.gs-webResult.gs-result a.gs-title:hover,
+
+.gs-webResult.gs-result a.gs-title:hover b {
```
Comment (mohawk2) [inline google-search.css:68]: I am happy with indentation changes, but extra blank lines here seem to make things less clear here, not more?

**Review by mohawk2 — CHANGES_REQUESTED** [review]
Thanks for this! A couple of tweaks.

**Review by dhdeangelis — COMMENTED** [review]
Fine, @mohawk2 ! Agree!

File: css/jquery-ui-1.8.2.custom.css (hunk 2 above)
Comment (mohawk2) [reply css/jquery-ui-1.8.2.custom.css:380]: Why is there a second `*/`? Is this code supposed to be removed?

File: content/home.html [hunk 4]
```diff
@@ -110,7 +110,7 @@ <h3><a href="https://sourceforge.net/p/pdl/news/">Recent News</a></h3>
 <script type="text/javascript">
 	// Start slideshow plugin.
 	$('.slideshow').cycle({
-		fx: 'fade,curtainX,zoom,turnDown,scrollRight',
+        fx: 'fade,curtainX,turnDown,scrollRight',
```
Comment (mohawk2) [inline content/home.html:113]: Wouldn't have mentioned only this, but there's the other point below. Reindenting this seems like not an improvement?

**Review by mohawk2 — CHANGES_REQUESTED** [review]
Thank you for the tweaks! Couple more questions I had.

**Review by dhdeangelis — COMMENTED** [review]
Changes made in local branch, added to my fork and (supposedly) pushed as PR

File: css/jquery-ui-1.8.2.custom.css (hunk 2 above)
Comment (dhdeangelis) [reply css/jquery-ui-1.8.2.custom.css:380]: I missed to respond to this. I had forgotten to delete the extra */ at the end of line 380. It is not anymore there, but I will wait until we solve this PR.

Comment (mohawk2) [reply css/jquery-ui-1.8.2.custom.css:380]: You say "it's not there anymore" but I can see it above. Can you help me understand what you mean?

Comment (dhdeangelis) [reply css/jquery-ui-1.8.2.custom.css:380]: I deleted from the local code and commited correctly in my fork. For some reason it is not seen here.

**Comment by mohawk2** [comment]
Closed in favour of #12.

**Pull request closed without merging**